src/variable.cpp

```diff
@@ -118,4 +131,42 @@ std::string Variable::getInitialValue() const
     return mPimpl->mInitialValue;
 }
 
+void Variable::setInterfaceType(const Variable::INTERFACE_TYPES interfaceType)
+{
+    mPimpl->mInterfaceType = interfaceType;
+}
+
+Variable::INTERFACE_TYPES Variable::getInterfaceType() const
+{
+    return mPimpl->mInterfaceType;
+}
+
+std::string interfaceTypeToString(Variable::INTERFACE_TYPES interfaceType)
+{
+    std::string str;
+    switch (interfaceType) {
+    case Variable::INTERFACE_TYPES::INTERFACE_TYPE_UNDEFINED: {
+        str = "";
+        break;
+    }
+    case Variable::INTERFACE_TYPES::INTERFACE_TYPE_NONE: {
+        str = "none";
+        break;
```

Comment:
The switch is set up this way to avoid a `control reaches end of non-void function` compile error: even if an `enum` is fully covered by the switch, it is possible to define an enumeration that has values not defined by any of its enumerators. So the `return` should be  outside of the enum cases unless we use a `default`.

This also follows existing practice in the codebase (see `prefixToString()` in `units.cpp`).